kas/base.yml

```diff
@@ -0,0 +1,54 @@
+header:
+    version: 19
+
+distro: nile
+
+defaults:
+    repos:
+        branch: "scarthgap"
+
+local_conf_header:
+    disk-space-monitoring: |
+        BB_DISKMON_DIRS ??= "\
+            STOPTASKS,${TMPDIR},1G,100K \
+            STOPTASKS,${DL_DIR},1G,100K \
+            STOPTASKS,${SSTATE_DIR},1G,100K \
+            STOPTASKS,/tmp,100M,100K \
+            HALT,${TMPDIR},100M,1K \
+            HALT,${DL_DIR},100M,1K \
+            HALT,${SSTATE_DIR},100M,1K \
+            HALT,/tmp,10M,1K"
+
+    ipk-packages: |
+        PACKAGE_CLASSES ?= "package_ipk"
+
+repos:
+    meta-nile:
+        path: "meta-nile"
+        layers:
+            .:
+    bitbake:
+        url: "https://github.com/openembedded/bitbake.git"
```

Comment:
[optional] Can we switch to using the official git.openembedded.org repos for bitbake, oe-core and meta-oe as the ones on github are mirrors? I know we've been using github mirros for submodules but if we're going to remove submodules once kas is fully working, we may as well make the change for kas now.